pkg/occlient/occlient_test.go

```diff
@@ -2460,3 +2461,69 @@ func TestCreateService(t *testing.T) {
 		})
 	}
 }
+
+func TestGetDeploymentConfigsFromSelector(t *testing.T) {
+	tests := []struct {
+		name     string
+		selector string
+		label    map[string]string
+		wantErr  bool
+	}{
+		{
+			name:     "true case",
+			selector: "app.kubernetes.io/name=app",
+			label: map[string]string{
+				"app.kubernetes.io/name": "app",
+			},
+			wantErr: false,
+		},
+		{
+			name:     "true case",
+			selector: "app.kubernetes.io/name=app1",
+			label: map[string]string{
+				"app.kubernetes.io/name": "app",
+			},
+			wantErr: false,
+		},
+	}
+
+	listOfDC := appsv1.DeploymentConfigList{
+		Items: []appsv1.DeploymentConfig{
+
+			{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app.kubernetes.io/name": "app",
+					},
+				},
+			},
+		},
+	}
+
+	// fmt.Println(tests)
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			fakeClient, fakeClientSet := FakeNew()
+
+			fakeClientSet.AppsClientset.PrependReactor("list", "deploymentconfigs", func(action ktesting.Action) (bool, runtime.Object, error) {
+				return true, &listOfDC, nil
+			})
+			dc, err := fakeClient.GetDeploymentConfigsFromSelector(tt.selector)
+
+			if len(fakeClientSet.AppsClientset.Actions()) != 1 {
+				t.Errorf("test failed")
+			}
+
+			if tt.wantErr == false && err != nil {
+				t.Errorf("test failed, %#v", dc[0].Labels)
```

Comment:
Same here something like with expected and current output https://github.com/redhat-developer/odo/blob/3f11739e4c29583844222dc2f904dd0d982c8f33/pkg/occlient/occlient_test.go#L1892